purchase_order_line_description/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Purchase order line description
+===============================
+
+This module allows to use only the product purchase description on the purchase
+order lines.
```

Comment:
@tafaRU please add the comparison with standard behaviour, to highlight the purpose of the module